code/game/machinery/vending.dm

```diff
@@ -938,43 +937,30 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/under/roll = 1,
 					/obj/item/clothing/head/clownpiece = 1,
 					/obj/item/clothing/under/clownpiece = 1,
-					/obj/item/clothing/suit/clownpiece = 1
-					)
-	contraband = list(/obj/item/clothing/suit/judgerobe = 1,
-					/obj/item/clothing/head/powdered_wig = 1,
-					/obj/item/weapon/gun/magic/wand = 2,
-					/obj/item/clothing/glasses/sunglasses/garb = 2,
-					/obj/item/clothing/gloves/anchor_arms = 1,
-					/obj/item/clothing/suit/kaminacape = 1,
-					/obj/item/clothing/under/soldieruniform = 1,
-					/obj/item/clothing/head/stalhelm = 1,
-					/obj/item/clothing/suit/soldiercoat = 1,
-					/obj/item/clothing/under/officeruniform = 1,
-					/obj/item/clothing/head/naziofficer = 1,
-					/obj/item/clothing/suit/officercoat = 1,
-					/obj/item/clothing/head/panzer = 1,
-					/obj/item/clothing/under/russobluecamooutfit = 1,
-					/obj/item/clothing/head/russobluecamohat = 1,
-					/obj/item/clothing/suit/russofurcoat = 1,
-					/obj/item/clothing/head/russofurhat = 1,
-					/obj/item/clothing/under/rottensuit = 1,
-					/obj/item/clothing/shoes/rottenshoes = 1,
-					/obj/item/clothing/head/helmet/biker = 1,
-					/obj/item/clothing/under/bikersuit = 1,
-					/obj/item/clothing/gloves/bikergloves = 1,
-					/obj/item/clothing/under/jacketsuit = 1,
-					/obj/item/clothing/head/helmet/richard = 1,
-					/obj/item/clothing/under/vault13 = 1
-					)
-	premium = list(/obj/item/clothing/suit/pirate/captain = 2,
-					/obj/item/clothing/head/pirate/captain = 2,
-					/obj/item/clothing/head/helmet/roman = 1,
-					/obj/item/clothing/head/helmet/roman/legionaire = 1,
-					/obj/item/clothing/under/roman = 1,
-					/obj/item/clothing/shoes/roman = 1,
-					/obj/item/weapon/shield/riot/roman = 1,
-					/obj/item/weapon/skub = 1
-					)
```

Comment:
You need to sync your branch with the life one, or just hard reset, because you're repeating the same changes already done to these files.